Cargo.toml

```diff
@@ -599,13 +599,74 @@ pedantic = { level = "deny", priority = -1 }
 unused_qualifications = "warn"
 
 [workspace.lints.clippy]
-all = { level = "deny", priority = -1 }
-#cargo = { level = "warn", priority = -1 }
-cognitive_complexity = "warn"
-default_trait_access = "warn"
-implicit_clone = "warn"
-manual_string_new = "warn"
-match_bool = "warn"
-range-plus-one = "warn"
-redundant-clone = "warn"
-ref_option = "warn"
+# The counts were generated with this command:
+#   cargo clippy --all-targets --workspace --message-format=json --quiet \
+#     | jq -r '.message.code.code | select(. != null and startswith("clippy::"))' \
+#     | sort | uniq -c | sort -h -r
+#
+# TODO:
+#  remove large_stack_arrays when https://github.com/rust-lang/rust-clippy/issues/13774 is fixed
+#
+all = { level = "warn", priority = -1 }
+cargo = { level = "warn", priority = -1 }
+pedantic = { level = "warn", priority = -1 }
+cargo_common_metadata = "allow"              # 3240
```

Comment:
why keeping these numbers? They will be outdated tomorrow

Reply:
@sylvestre I use these numbers to guide which one I fix next - at which point I remove the entire line, not just the comment